tests/Feature/User/MakeUserActiveTest.php

```diff
@@ -21,12 +21,14 @@ class MakeUserActiveTest extends TestCase
     protected function setUp(): void
     {
         parent::setUp();
-        $this->admin = $admin = factory(User::class)->create();
+        // $this->admin = $admin = factory(User::class)->create();
+        $this->admin = $admin = User::factory()->create();
 
         $admin->roles()->attach(Config::get('constants.Administrator'));
         $this->be($admin);
 
-        $this->user = $user = factory(User::class)->create();
+        // $this->user = $user = factory(User::class)->create();
```

Comment:
Why is this commented out?? it seems the notation has changed? is this line still needded or can it be deleted? Please also check the different test files